### PR TITLE
chore: NO-JIRA add pull request template

### PR DESCRIPTION
**Pull request by mjamrozekvl**
I added pull request template because we would like to have checkboxes that remind us about necessary actions that have to be done. First important action is to export component that we implement in global `src/index.ts` file.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## What did you implement?
+
+_A summary of the change from the implementation perspective that could help
+reviewers understanding the whole solution_
+
+## Checklist:
+
+- [ ] I exported my component in [src/index.ts](https://github.com/VirtusLab/tetrisly-react/blob/main/src/index.ts) file.
```
Comment (mwleklinski) [inline .github/PULL_REQUEST_TEMPLATE.md:8]: I would add the following checks: 
```suggestion
- [ ] I exported my component in [src/index.ts](https://github.com/VirtusLab/tetrisly-react/blob/main/src/index.ts) file.
- [ ]  I described colors, spacing, typography, and similar attributes with Figma Tokens and they are placed in the config object in the `*.styles` file
- [ ] I tested the component (including custom prop)
- [ ] I've written a storybook story for the component
```

@adrian-potepa would you add anything?

**Review by mwleklinski — COMMENTED** [review]
Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (adrian-potepa) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Yep, it will be awesome 👌

**Review by adrian-potepa — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (mjamrozekvl) [reply .github/PULL_REQUEST_TEMPLATE.md:8]: Ok, I added suggested checkboxes ;-)

**Pull request merged**